.mage/docs.go

```diff
@@ -47,6 +73,14 @@ func (d Docs) yarn() (func(args ...string) error, error) {
 
 // Deps installs the documentation dependencies.
 func (d Docs) Deps() error {
+	fileUrl := "https://raw.githubusercontent.com/TheThingsNetwork/lorawan-frequency-plans/master/frequency-plans.yml"
+	error := downloadFile("doc/data/frequency-plans.yml", fileUrl)
+	if error != nil {
+		panic(error)
+	}
+	if mg.Verbose() {
+		fmt.Println("Downloaded: " + fileUrl)
```

Comment:
In verbose mode we should actually be verbose, and say exactly what we're doing. Something like

```suggestion
		fmt.Printf("Downloaded %q to %q\n", url, target)
```